cpp/ql/src/semmle/code/cpp/Declaration.qll

```diff
@@ -119,6 +119,19 @@ abstract class Declaration extends Locatable, @declaration {
   /** Holds if this declaration has the given name in the global namespace. */
   predicate hasGlobalName(string name) { this.hasQualifiedName("", "", name) }
 
+
+  /** Holds if this declaration has the given name in the `std` namespace. */
+  predicate hasStdName(string name) {
+    this.hasQualifiedName("std", "", name)
```

Comment:
This can be written as `this.hasQualifiedName("std", name)`. But is it worth adding a public predicate for this? It's on `Declaration`, so it'll show up in a lot of autocomplete lists.

Reply:
Given how little it's actually used, probably not.